src/dbhelpers.h

```diff
@@ -16,6 +16,10 @@
 #define ATTRIB_SENTINEL
 #endif
 
+#if defined(SOLARIS)
+#define __STDC_WANT_LIB_EXT1__ 1
```

Comment:
I think this can be enabled unconditionally, no need to check for Solaris. AFAIK the only reason it's hidden behind a `#define` is in case of clashes with program symbols, but that won't be the case here.

Reply:
I'm pretty sure it'll be fine without the `#if defined(SOLARIS)`